src/main/app/claims/models/claim.ts

```diff
@@ -159,6 +159,8 @@ export class Claim {
       return ClaimStatus.CLAIMANT_ACCEPTED_COURT_PLAN_SETTLEMENT
     } else if (this.isSettlementReached()) {
       return ClaimStatus.OFFER_SETTLEMENT_REACHED
+    } else if (this.hasClaimantRejectedDefendantDefenceWithoutDQs()) {
+      return ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ
```

Comment:
Do we need to handle `CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE_NO_DQ` scenario?

Reply:
that's covered by hasClaimantAcceptedDefendantDefence

Reply:
ok